cmd/torrent-verify/main.go

```diff
@@ -16,16 +16,40 @@ import (
 )
 
 var (
-	filePath = flag.String("torrent", "/path/to/the.torrent", "path of the torrent file")
-	dirPath  = flag.String("path", "/torrent/data", "path of the torrent data")
+	torrentPath = flag.String("torrent", "/path/to/the.torrent", "path of the torrent file")
+	dataPath    = flag.String("path", "/torrent/data", "path of the torrent data")
```

Comment:
Currently torrent-verify works by verifying the data in the current working directory. The OS already has a provision for setting the working directory, I think adding another flag just complicates the the cmd's interface. A doc note saying "data is read from the working directory" could be better.